fili-generic-example/README.md

```diff
@@ -13,6 +13,15 @@ Note that this was last tested using [version 0.9.1](https://github.com/yahoo/fi
 
 ## Setup and Launching
 
+### Docker
+There is a Fili Dockerfile which is available on Dockerhub. To start playing around with Fili, you can download and open Docker. Then
```

Comment:
Should probably add a link to the page on docker hub.

Also maybe add a link to the [docker install page](https://www.docker.com/community-edition)